ddp-workspace/projects/ddp-dsm-ui/src/app/add-patients-modal/add-patients-modal.component.html

```diff
@@ -0,0 +1,67 @@
+<p>Add Patient(s)</p>
+<p>
+  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo itaque vel
+  labore quasi numquam sapiente quidem a corporis, nobis inventore veritatis
+  minus rem similique praesentium. Consequuntur, ab tenetur! Quis, ab.
+</p>
```

Comment:
[Our mockup](https://dspuxteam.invisionapp.com/console/Center-view-cl2j0jldt0e9d01b1b0i2bgrw/cl4d590gt0a4h01zehk242bpm/play) does not have that paragraph.